multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2004,6 +2005,142 @@ def save(self, path: str, standalone: Optional[bool] = True):
                     f"removed."
                 )
 
+    def export_onnx(
+        self,
+        onnx_path: Optional[str] = None,
+        data: Optional[pd.DataFrame] = None,
+        batch: Optional[Dict] = None,
+        batch_size: Optional[int] = None,
+        verbose: Optional[bool] = False,
+        opset_version: Optional[int] = 13,
+    ):
+        """
+        Export this predictor's model to ONNX file.
+
+        Parameters
+        ----------
+        onnx_path
+            The export path of onnx model.
+        data
+            Raw data used to trace and export the model.
+            If this is None, will check if a processed batch is provided.
+        batch
+            Processed batch used to trace and export the model. It is used if data is None.
+            If this is also None, a default batch will be used.
+        batch_size
+            The batch_size of export model's input.
+            Normally the batch_size is a dynamic axis, so we could use a small value for faster export.
+        verbose
+            verbose flag in torch.onnx.export.
+        opset_version
+            opset_version flag in torch.onnx.export.
+        """
+        # TODO: Support CLIP
+        # TODO: Add test
+
+        valid_input, dynamic_axes, default_onnx_path, default_batch = get_onnx_input(
+            pipeline=self._pipeline, config=self._config
+        )
+        if not onnx_path:
+            onnx_path = default_onnx_path
+
+        model = self._model
+        model.eval()
```

Comment:
How about removing these two lines and directly pass `self._model.eval()` to `torch.onnx.export()`?

Reply:
Done, thanks!